tests/IceRpc.Locator.Tests/LocationResolverTests.cs

```diff
@@ -23,7 +23,7 @@ public class LocationResolverTests
                 TimeSpan.FromSeconds(refreshThreshold),
                 ttl: Timeout.InfiniteTimeSpan);
 
-        (ServiceAddress? resolved, bool _) = await resolver.ResolveAsync(new Location(), refreshCache: true, default);
+        (ServiceAddress? resolved, bool _) = await resolver.ResolveAsync(default, refreshCache: true, default);
```

Comment:
How does this work for the required property `Value`?

Reply:
For structs, "default" bypasses the "required" check, while "new T()" doesn't.

See https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-11.0/required-members#struct-defaults